Code/UnitTests/TestFramework/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
 ez_cmake_init()
 
+ez_build_filter_foundation()
+
 set(EZ_TESTFRAMEWORK_USE_FILESERVE OFF)
 
-if(EZ_CMAKE_PLATFORM_WINDOWS_UWP)
+get_property(filterIndex GLOBAL PROPERTY EZ_BUILD_FILTER_INDEX)
+if(EZ_CMAKE_PLATFORM_WINDOWS_UWP AND filterIndex GREATER_EQUAL 1)
```

Comment:
Can we make this more pretty?

Reply:
I thought about this quite some time, but couldn't find a way to make it more pretty. I'm open to suggestions.

Reply:
As far as I understand it, this code checks whether the "everything" filter is set, right? Could that just be put into a macro to hide the implementation detail and give it a descriptive name?